docs/_static/nidigital_usage.inc

```diff
@@ -1,27 +1,45 @@
 Usage
 ------
 
-The following is a basic example of using the **nidigital** module to open a session to a ....
+The following is a basic example of using the **nidigital** module to open a session to a digital pattern instrument
+and source/measure both voltage and current using the PPMU on selected channels.
 
 .. code-block:: python
 
     import nidigital
-    with nidigital.Session(resource_name='PXI1Slot2', channels='0') as session:
-        pass
+    import time
 
+    with nidigital.Session(resource_name='PXI1Slot2') as session:
```

Comment:
Should we have simulation enabled by default in this code snippet? It would allow the user to run the code snippet without any actual hardware? Other MI driver usage file do not show simulation.

Reply:
No, we should do like other drivers. Why do you propose to be different / inconsistent?

Reply:
It seems like the code snippet would be more useful if it can run without hardware, but doing without simulation like other MI driver is also okay.

One thing I have noticed is that the amount of content in the usage files across the different MI drivers is very inconsistent.

Reply:
>It seems like the code snippet would be more useful if it can run without hardware, but doing without simulation like other MI driver is also okay.

Most customers trying this stuff out have hardware.

> One thing I have noticed is that the amount of content in the usage files across the different MI drivers is very inconsistent.
 
Good observation, PRs welcome!